aspoet/src/main/kotlin/com/google/androidstudiopoet/generators/android_modules/AndroidModuleBuildBazelGenerator.kt

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2018 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ */
+
+package com.google.androidstudiopoet.generators.android_modules
+
+import com.google.androidstudiopoet.models.*
+import com.google.androidstudiopoet.writers.FileWriter
+
+class AndroidModuleBuildBazelGenerator(val fileWriter: FileWriter) {
```

Comment:
What do you think, would it make sense to have an analog for `GradleLang.kt.` for Bazel? Then support and modification of Bezel-related files might be easier.

Reply:
https://github.com/android/android-studio-poet/pull/135